Makefile

```diff
@@ -0,0 +1,11 @@
+DOCKER_BIN = docker
+DOCKER_IMAGE = ubuntu-focal
```

Comment:
Add TAR_BIN and use it as variable in dockerfile, similar to DOCKER_BIN

Reply:
done